stakepooldclient/stakepooldclient.go

```diff
@@ -265,6 +266,48 @@ func (s *StakepooldManager) VoteVersion() (uint32, error) {
 	return lastVersion, nil
 }
 
+// ValidateAddress calls ValidateAddress RPC on all stakepoold servers.
+// Returns an error if responses are not the same from all stakepoold instances.
+func (s *StakepooldManager) ValidateAddress(addr dcrutil.Address) (*pb.ValidateAddressResponse, error) {
+	responses := make(map[int]*pb.ValidateAddressResponse)
+
+	// Get ValidateAddress response from all wallets
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		req := &pb.ValidateAddressRequest{
+			Address: addr.EncodeAddress(),
+		}
+		resp, err := client.ValidateAddress(context.Background(), req)
+		if err != nil {
+			log.Errorf("ValidateAddress RPC failed on stakepoold instance %d: %v", i, err)
+			return nil, err
+		}
+		responses[i] = resp
+	}
+
+	// Ensure responses are identical
+	var lastResponse *pb.ValidateAddressResponse
+	lastServer := 0
```

Comment:
Just a style thing - at least in dcrdata's codebase, we always init with `var name type` when the zero value is desired.  I'm not sure how consistent dcrstakepool is with variable initialization though.